### PR TITLE
Add accessibility statement

### PR DESCRIPTION
**Pull request by allbecauseyoutoldmeso**
### What

- Add statement at '/accessibility'
- Add footer link to page

### Storycard

https://trello.com/c/yxsCI1y6/1019-add-accessibility-statement-to-bops-applicant-footer

### Screenshots

Statement:

Footer link:

### CODE REVIEW

File: spec/system/accessibility_page_spec.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        email_address: "planning@default.gov.uk",
+        phone_number: "01234123123",
+        postal_address: "Planning, 123 High Street, Big City, AB3 4EF",
+      },
+    }.deep_stringify_keys
+  end
+
+  before do
+    ENV["API_BEARER"] = "123"
+    ENV["PROTOCOL"] = "https"
```
Comment (benbaumann95) [inline spec/system/accessibility_page_spec.rb:16]: Out of scope for this PR but reminds me that we should extract this horrible thing as it's in every spec

File: config/locales/en.yml [hunk 2]
```diff
@@ -34,6 +34,58 @@ en:
       the_following_request: The following request has been made.
       title: Respond to other request
       what_you_need: "What you need to do:"
+  pages:
```
Comment (benbaumann95) [inline config/locales/en.yml:37]: 🎉

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -8,4 +8,20 @@ module ApplicationHelper
   def feedback_email(current_local_authority)
     FEEDBACK_EMAILS[current_local_authority.to_sym]
   end
+
+  def formatted_phone_number
+    number_to_phone(
```
Comment (benbaumann95) [inline app/helpers/application_helper.rb:13]: Didn't know about this helper 👀

**Review by benbaumann95 — APPROVED** [review]

File: spec/system/accessibility_page_spec.rb (hunk 1 above)
Comment (allbecauseyoutoldmeso) [reply spec/system/accessibility_page_spec.rb:16]: What do you think about using Climate Control? https://github.com/thoughtbot/climate_control

**Pull request merged**